src/component/selector.js

```diff
@@ -320,6 +320,12 @@ export default class Selector {
     setAllAreaOffset.call(this, this.data.getSelectedRect());
   }
 
+  setStartEnd(sri, sci, eri, eci, mri, mci) {
```

Comment:
@aleschneck Can these variables be extended to give some meaning? I have no idea what `sri`, `eci` etc means. Or a brief comment to explain